code/src/components/AddTasks.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components/macro';
+import { Wrapper, Actionbtn } from 'styles/GlobalStyles';
+import { tasks } from 'reducers/tasks';
+import plusSign from 'assets/plusGreen.png';
+
+export const AddTasks = () => {
+  const [newTask, setNewTask] = useState(false);
+  const [message, setMessage] = useState('');
+
+  const dispatch = useDispatch();
+
+  // Prevent form from refreshing whole page when submitted
+  const onFormSubmit = (event) => {
+    event.preventDefault()
+  };
+
+  // Function for adding task to the store
+  const sendTaskToStore = (item) => {
+    if (item.length > 0) {
+      dispatch(tasks.actions.addToDo(item))
+    }
+    setMessage('')
+  };
+
+  // Function for storing user's text
+  const handleChange = (event) => {
+    setMessage(event.target.value);
+  };
+
+  return (
+    // When no task is being added, display a plus sign for entering a new task
+    // When the plus sign is clicked, display a text input field for new task
+    <>
+      {!newTask && (
+        <AddTaskWrapper>
+          <AddBtn
+            type="button"
+            onClick={() => setNewTask(true)}>
+            <img src={plusSign} alt="" />
+          </AddBtn>
+        </AddTaskWrapper>
+      )}
```

Comment:
Nicely done! I like the plus image and that you can close and remove the form! Inspiring!